packages/analytics-js/src/state/slices/consents.ts

```diff
@@ -1,17 +1,16 @@
 import { Signal, signal } from '@preact/signals-core';
-import { CookieConsentOptions } from '@rudderstack/analytics-js/state/types';
 
 export type ConsentsState = {
   deniedConsentIds: Signal<string[]>;
   allowedConsentIds: Signal<string[]>;
-  cookieConsentOptions: Signal<CookieConsentOptions>;
+  consentManager: Signal<string | undefined>;
```

Comment:
Can we make it instead store the enum value?

Reply:
To avoid calculating multiple times we are storing it. That's what we discussed. So that we can use only this value to validate if a valid consentManager has been provided.

Reply:
The parameter can be marked as optional to allow that.